### DISCUSSION TITLE
最近评论能否选择性开启

### DISCUSSION THREAD

**Discussion by ycloong** [tangly1024/NotionNext#303] [Q&A]
如题，因为我不怎么需要最近评论这个模块，但是我有不会怎么改，所以想球球作者了，可不可以把它配置成为可选配置？

**Comment by tangly1024** [comment]
可以，已列入开发计划 https://github.com/tangly1024/NotionNext/issues/304

**Comment by tangly1024** [comment]
- 在最新的[v3.4.1版本](https://github.com/tangly1024/NotionNext/releases/tag/v3.4.1)，默认已关闭“最近评论”模块；
- 开启最新评论可参考：https://github.com/tangly1024/NotionNext/issues/304 描述。
